unstructured_inference/inference/layoutelement.py

```diff
@@ -237,7 +241,9 @@ def table_cells_to_dataframe(cells: dict, nrows: int = 1, ncols: int = 1, header
     return DataFrame(arr, columns=header)
 
 
-def partition_groups_from_regions(regions: Collection[TextRegion]) -> List[List[TextRegion]]:
+def partition_groups_from_regions(
+    regions: Collection[TextRegion],
+) -> List[List[TextRegion]]:
```

Comment:
This looks like it maybe shouldn't have been reformatted.

Reply:
Fixed, thanks!